applications/GeoMechanicsApplication/custom_strategies/schemes/geomechanics_time_integration_scheme.hpp

```diff
@@ -122,7 +122,7 @@ class GeoMechanicsTimeIntegrationScheme : public Scheme<TSparseSpace, TDenseSpac
         Scheme<TSparseSpace, TDenseSpace>::Initialize(rModelPart);
 
         KRATOS_TRY
-        SetTimeFactors(rModelPart);
+        // SetTimeFactors(rModelPart); commented to avoid initialization of mDeltaTime when the value is not available yet
```

Comment:
This can be completely deleted, including the TRY CATCH structure.

Reply:
I kept the comment to avoid a code smell. You are absolutely right that everything can be deleted. Hope there are no any code smells.